resources/authgear/templates/en/web/authflowv2/whatsapp_otp.html

```diff
@@ -0,0 +1,80 @@
+{{ template "authflowv2/__page_frame.html" . }}
+{{ define "page-content" }}
+
+  {{ $has_alternatives := false }}
+	{{ range $.IdentityCandidates }}
+		{{ if eq .type "oauth" }}
+      {{ $has_oauth = true }}
+      <!-- TODO(tung): Add SSO (https://github.com/authgear/authgear-server/issues/3593) -->
+      {{/* $has_alternatives = true */}}
+		{{ end }}
+		{{ if eq .type "login_id" }}
+		  {{ $has_login_id = true }}
+      {{ if not (eq .login_id_key $.LoginIDKey) }}
+		    {{ $has_alternatives = true }}
+      {{ end }}
+		{{ end }}
+	{{ end }}
+
+  {{ if $.Error }}
+    {{ if not (eq $.Error.reason "InvalidCredentials") }}
+      <div class="mb-8 alert-message alert-message--error">
+        <span class="alert-message__icon material-icons material-icons--filled">
+          error
+        </span>
+        <span class="alert-message__text">
+          {{ template "authflowv2/__error.html" . }}
+        </span>
+      </div>
+    {{ end }}
+  {{ end }}
+
+  <div class="space-y-4">
+    {{ template "authflowv2/__header.html" . }}
+    <h1 class="screen-title">
+      {{ template "v2-whatsapp-otp-page-title" }}
+    </h1>
+    <h2 class="screen-description">
+      {{ template "v2-whatsapp-otp-page-subtitle" . }}
+    </h2>
+  </div>
+  <div class="mt-8">
+    {{ $otp_input_error_context := false }}
+    {{ if $.Error  }}
+      {{ if eq $.Error.reason "InvalidCredentials" }}
+        {{ $otp_input_error_context = $ }}
+      {{ end }}
+    {{ end }}
+
+    <!--
+      FIXME(newman): Use composable template for label props `ResendButtonLabel` and `ResendButtonLabelWithValue`
```

Comment:
Maybe you can try to cherry pick the commit in https://github.com/authgear/authgear-server/pull/3646